### PR TITLE
(PC-11993) feat(ubble): enable redirection to ubble on web

### PR DESCRIPTION
**Pull request by antoinewg**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-11993

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/identityCheck/pages/identification/IdentityCheckWebview.web.tsx [hunk 1]
```diff
@@ -1,16 +1,64 @@
...
-import { GenericInfoPage } from 'ui/components/GenericInfoPage'
-import { Error } from 'ui/svg/icons/Error'
-import { getSpacing } from 'ui/theme'
+import { REDIRECT_URL_UBBLE } from 'features/identityCheck/api'
+import { UseNavigationType, UseRouteType } from 'features/navigation/RootNavigator'
+import { Helmet } from 'libs/react-helmet/Helmet'
+import { analytics } from 'libs/analytics'
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+declare const Ubble: any
```
Comment (BenStirrup) [inline src/features/identityCheck/pages/identification/IdentityCheckWebview.web.tsx:10]: C'est pk ca ?

File: src/features/identityCheck/pages/identification/IdentityCheckWebview.web.tsx [hunk 2]
```diff
@@ -1,16 +1,64 @@
...
+        },
+        onAbort({ redirectUrl, status, returnReason }: AbortEvent) {
+          analytics.logIdentityCheckAbort({ status, returnReason })
+          ubbleIDV.destroy()
+          // TODO(antoinewg): navigate to an error page.
+          if (redirectUrl.includes(REDIRECT_URL_UBBLE)) navigate('IdentityCheckEnd')
+        },
+      },
+    })
+  }
```
Comment (BenStirrup) [inline src/features/identityCheck/pages/identification/IdentityCheckWebview.web.tsx:49]: faut pas mettre cela dans un `useEffect` ?

File: src/features/identityCheck/pages/identification/IdentityCheckWebview.web.tsx (hunk 1 above)
Comment (antoinewg) [reply src/features/identityCheck/pages/identification/IdentityCheckWebview.web.tsx:10]: parce que j'utilise `Ubble` après et typescript est pas content sinon car il le reconnait pas.

File: src/features/identityCheck/pages/identification/IdentityCheckWebview.web.tsx (hunk 2 above)
Comment (antoinewg) [reply src/features/identityCheck/pages/identification/IdentityCheckWebview.web.tsx:49]: Allé 👍

**Review by Lucasbeneston — APPROVED** [review]
👍

**Pull request merged**